vault/logical_system.go

```diff
@@ -2841,6 +2841,7 @@ func (b *SystemBackend) pathHashWrite(ctx context.Context, req *logical.Request,
 
 func (b *SystemBackend) pathRandomWrite(ctx context.Context, req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
 	bytes := 0
+	const maxBytes = 128 * 1024
```

Comment:
Same comment

Reply:
Solved in https://github.com/hashicorp/vault/pull/7144/commits/2699e4ea2abc9282ac263687148902a704d6bc4e